openwisp_radius/api/freeradius_views.py

```diff
@@ -237,3 +350,46 @@ def perform_create(self, serializer):
 
 
 postauth = PostAuthView.as_view()
+
+
+class UserAccountingFilter(AccountingFilter):
+    class Meta(AccountingFilter.Meta):
+        fields = [
+            field for field in AccountingFilter.Meta.fields if field != 'username'
+        ]
+
+
+@method_decorator(
+    name='get',
+    decorator=swagger_auto_schema(
+        operation_description="""
+        **Requires the user auth token (Bearer Token).**
+        Returns the radius sessions of the logged-in user and the organization
+        specified in the URL.
+        """,
+    ),
+)
+class UserAccountingView(ThrottledAPIMixin, DispatchOrgMixin, ListAPIView):
```

Comment:
this one doesn't go here, because it's not a freeradius endpoint but merely an API which shows the freeradius sessions (RadiusAccounting records) of the logged in user, designed to be used from openwisp-wifi-login-pages to show the historic sessions of the logged in user.

Do you see that this endpoint does not use `FreeradiusApiAuthentication`?

Reply:
I made the change here because it was fetching data from `RadiusAccounting`. 
Thanks for the explanation, will revert it soon